### PR TITLE
update rails, change icons, add guest links search, change guest link view

### PR DESCRIPTION
**Pull request by pavlozhuravlov**
Links view for guest
image

Links view for authenticated user
image

Search result for guest
image

Icons 
image

### CODE REVIEW

**Comment by tilast** [comment]
@rilian plz consider adding open-source travis here or move it to gitlab

File: app/views/links/index.slim [hunk 1]
```diff
@@ -1,48 +1,81 @@
...
           th Issue
-          th.actions Actions
+          - if manager_signed_in?
+            th.actions Actions
       tbody
         - @links.each do |link|
           tr
-            td #{link.id}
+            - if manager_signed_in?
+             td #{link.id}
```
Comment (tilast) [inline app/views/links/index.slim]: I guess you've missed on space before `td`

File: app/views/links/index.slim [hunk 2]
```diff
@@ -1,48 +1,81 @@
...
+    /                a href="#{links_path(tag: tag)}" #{tag}
+    /            span #{link.time_to_read.present? && link.time_to_read > 0 ? " (#{link.time_to_read} mins reading)" : ''}
+    /            p.link-desc
+    /              |  #{link.desc.strip.gsub("\n", '<br/>').html_safe}  
+    /          td
+    /            - if link.issue.present?
+    /              a href="#{issue_path(link.issue)}" ##{link.issue_id}
+    /            - else
+    /              span
+    /      = paginate @links
```
Comment (tilast) [inline app/views/links/index.slim]: why is this code commented?

File: .semaphore-cache/rubocop_cache/rubocop_cache/1567e4615c9958245b9099e70815f129/_/ffe1ebd764eac00596ea3b511eeb167c [hunk 3]
```diff
@@ -0,0 +1 @@
+[]
```
Comment (tilast) [inline .semaphore-cache/rubocop_cache/rubocop_cache/1567e4615c9958245b9099e70815f129/_/ffe1ebd764eac00596ea3b511eeb167c]: plz remove cache files

**Review by tilast — CHANGES_REQUESTED** [review]

**Review by tilast — APPROVED** [review]

File: app/views/links/index.slim [hunk 4]
```diff
@@ -1,48 +1,58 @@
...
                 |  (#{link.time_to_read} mins reading)
-              br
+              
               = link.desc.strip.gsub("\n", '<br/>').html_safe
-              - if link.manager
-              br
-                | by #{link.manager.email.split('@').first}
+              - if manager_signed_in?
+                br
+                  | by #{link.manager.email.split('@').first}
```
Comment (rilian) [inline app/views/links/index.slim:48]: .manager can be nil

File: spec/controllers/links/index_spec.rb [hunk 5]
```diff
@@ -29,24 +29,11 @@
           issue.update(sent_at: Time.zone.now)
         end
 
-        it 'renders search_by_tag template' do
-          get :index, tag: 'first_tag'
-
-          expect(assigns(:links)).to include first_link
-          expect(assigns(:links)).not_to include second_link
```
Comment (rilian) [inline spec/controllers/links/index_spec.rb:36]: why the test is removed?

File: spec/services/link_filter_spec.rb [hunk 6]
```diff
@@ -60,9 +60,7 @@
     context 'without tag param' do
       let(:params) { {} }
 
-      it 'returns empty search by tag result' do
-        expect(result).to eq []
```
Comment (rilian) [inline spec/services/link_filter_spec.rb:64]: why the test is removed?

**Review by rilian — CHANGES_REQUESTED** [review]

**Comment by pavlozhuravlov** [comment]
plz review

File: spec/controllers/links/index_spec.rb [hunk 7]
```diff
@@ -29,11 +29,21 @@
           issue.update(sent_at: Time.zone.now)
         end
 
-        it 'renders index template'
+        it 'renders index template' do
+          get :index, tag: 'test'
+
```
Comment (tilast) [inline spec/controllers/links/index_spec.rb:34]: plz check `@links` contains something

File: spec/controllers/links/index_spec.rb [hunk 8]
```diff
@@ -29,11 +29,21 @@
...
+          expect(response).to render_template(:index)
+          expect(response.status).to eq 200
+        end
       end
 
       context 'when tag param is not specified' do
-        it 'renders index template'
+        it 'renders index template' do
+          get :index, tag: 'test'
+
```
Comment (tilast) [inline spec/controllers/links/index_spec.rb:45]: same here

**Review by tilast — CHANGES_REQUESTED** [review]

**Review by tilast — APPROVED** [review]

File: config/secrets.yml [hunk 9]
```diff
@@ -5,16 +5,16 @@
...
 # Make sure the secrets in this file are kept private
 # if you're sharing your code publicly.
 
 development:
-  secret_key_base: <%= ENV["SECRET_KEY_BASE"] %>
+  secret_key_base: 6df234370e39bc9f3b87566a28852b74fb5043959c6f25766aa64047aa3dfd43d6c8041b41a0f166916a06d81620753ff352f433492c8603bcd2a8d4f7c7fe5e
 
 test:
-  secret_key_base: <%= ENV["SECRET_KEY_BASE"] %>
+  secret_key_base: 11d2ab90ec2c6d0a00e25ece0df26703ec6ec00b8f4c5602e2f4f26fd8ac3529d46c9f85bec1613d0b92454c2d54e443a441cbd5e25f07cbc40b0abbdd027728
```
Comment (tilast) [inline config/secrets.yml]: why?

**Review by tilast — CHANGES_REQUESTED** [review]

**Comment by pavlozhuravlov** [comment]
plz review

**Pull request merged**